frameworks/mlr3automl/setup.sh

```diff
@@ -41,9 +41,11 @@ fi
 LIB="${HERE}/lib/"
 mkdir "${LIB}"
 
-Rscript -e 'options(install.packages.check.source="no"); install.packages(c("mlr3", "mlr3pipelines", "mlr3misc", "mlr3oml", "mlr3hyperband", "mlr3tuning", "paradox"), repos="https://cloud.r-project.org/", lib="'"${LIB}"'")'
+Rscript -e 'options(install.packages.check.source="no"); install.packages(c("mlr3", "mlr3pipelines", "mlr3misc", "mlr3oml", "mlr3tuning", "mlr3learners", "paradox"), repos="https://cloud.r-project.org/", lib="'"${LIB}"'")'
```

Comment:
some thing I tried in the past and worked well for me was to completely remove those explicits imports and use instead only:
```bash
Rscript -e 'options(install.packages.check.source="no"); install.packages(c("remotes"), repos="https://cloud.r-project.org/", lib="'"${LIB}"'")'
Rscript -e '.libPaths("'"${LIB}"'"); remotes::install_github("'"${REPO}"'", ref="'"${VERSION}"'", lib="'"${LIB}"'", dependencies=TRUE)'
Rscript -e '.libPaths("'"${LIB}"'"); remotes::install_github("'"${MLR_REPO}"'/mlr3extralearners", lib="'"${LIB}"'")'
```
I can confirm that it also install automatically `mlr3tuning`, `mlr3learners` and everything needed.

of course, if `mlr3hyperband` and `bbotk` needs to be installed from a custom repo, you can install them at the end using `upgrade='always'` to ensure it overrides the default library.
```bash
Rscript -e '.libPaths("'"${LIB}"'"); remotes::install_github("'"${MLR_REPO}"'/mlr3extralearners", lib="'"${LIB}"'", upgrade="'"always"'")'
Rscript -e '.libPaths("'"${LIB}"'"); remotes::install_github("'"${MLR_REPO}"'/mlr3hyperband", lib="'"${LIB}"'", upgrade="'"always"'")'
Rscript -e '.libPaths("'"${LIB}"'"); remotes::install_github("'"${MLR_REPO}"'/bbotk", lib="'"${LIB}"'", upgrade="'"always"'")'
```
If we're able to get rid of the listing of all dependencies and rely on the resolution provided by the package itself, it should make this setup much more robust. wdyt?

Reply:
This is a good idea.
mlr3automl currently links to the CRAN version of the dependencies, I'll have to make it so the dev versions are installed from github instead. I'll have a look tonight

Reply:
Done